src/MathOptFormat.jl

```diff
@@ -137,63 +136,88 @@ function create_unique_variable_names(
 end
 
 """
-List of accepted export formats. `AUTOMATIC_FILE_FORMAT` corresponds to
-a detection from the file name, only based on the extension (regardless of
-compression format).
+    FileFormat
+
+List of accepted export formats.
+
+`AUTOMATIC_FILE_FORMAT` corresponds to a detection from the file name, only
+based on the extension (regardless of compression format).
 """
-@enum(FileFormat, FORMAT_CBF, FORMAT_LP, FORMAT_MOF, FORMAT_MPS, AUTOMATIC_FILE_FORMAT)
+@enum(
+    FileFormat,
+    FORMAT_CBF,
+    FORMAT_LP,
+    FORMAT_MOF,
+    FORMAT_MPS,
+    AUTOMATIC_FILE_FORMAT,
+)
 
-const _file_formats = Dict{FileFormat, Tuple{String, Any}}(
+const _FILE_FORMATS = Dict{FileFormat, Tuple{String, Any}}(
     # ENUMERATED VALUE => extension, model type
     FORMAT_CBF => (".cbf", CBF.Model),
     FORMAT_LP => (".lp", LP.Model),
     FORMAT_MOF => (".mof.json", MOF.Model),
     FORMAT_MPS => (".mps", MPS.Model)
 )
 
-function _filename_to_format(filename::String)
```

Comment:
This function is used in the PR on JuMP's side: https://github.com/JuliaOpt/JuMP.jl/pull/1982/files#diff-8d49312dff85b4e2f1ab3e617c3e74e3R24

Reply:
It can be re-defined on the JuMP side.